src/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+export const getAllArticles = async () => {
+  const response = await axios.get(
+    "https://project-x8zn.onrender.com/api/articles"
```

Comment:
Very good to see this extracted out to an exportable function 👏 
Remember that as you add more of these functions you can create an _axios instance_ that will hold the main host URL